src/components/EditorRightSideBar/index.tsx

```diff
@@ -21,7 +21,7 @@ const EditorRightSideBar = () => {
               flex={item.id === itemNum ? 2 : 1}
               h={`${100 / (itemNum + 1)}%`}
               color="#FFFFFF"
-              p="20px 16px"
+              p="16px 10px"
```

Comment:
I think there is an issue with the horizontal and vertical margins. According to the UI display, it should be padding-top: 20px and padding-left: 16px. If you wish to adjust the padding to achieve the desired effect, it should be done in the "FileManagement" file settings rather than directly modifying the overall layout.